include/cctz/civil_time_detail.h

```diff
@@ -79,14 +79,16 @@ CONSTEXPR_F bool is_leap_year(year_t y) noexcept {
   return y % 4 == 0 && (y % 100 != 0 || y % 400 == 0);
 }
 CONSTEXPR_F int year_index(year_t y, month_t m) noexcept {
-  return (static_cast<int>((y + (m > 2)) % 400) + 400) % 400;
+  int year_index = static_cast<int>((y + (m > 2)) % 400);
+  if (year_index < 0) {
```

Comment:
I guess I would also prefer to see this as a conditional return rather than a conditional increment, so ...
```
CONSTEXPR_F int year_index(year_t y, month_t m) noexcept {
  int yi = static_cast<int>((y + (m > 2)) % 400);
  return yi < 0 ? yi + 400 : yi;
}
```

Reply:
Done